sphinx_needs/lsp/esbonio.py

```diff
@@ -84,9 +111,13 @@ def hover(self, context: HoverContext) -> str:
         self.logger.debug(f"hover params: {context}")
 
         if isinstance(self.rst, SphinxLanguageServer) and self.rst.app:
+            # get and check needs.json path
+            needs_json = get_needs_json(self.rst.app)
+            if not needs_json or not needs_json.exists():
```

Comment:
you can use one of de Morgan's laws here

```suggestion
            if not (needs_json and needs_json.exists()):
```